src/components/select/SelectReps/index.tsx

```diff
@@ -73,42 +73,67 @@ const SelectReps: React.FC<SelectRepsProps> = (props) => {
       <FormLabel sx={{ display: "flex", alignItems: "center", gap: 1 }}>
         Reps: {props.reps}
         {props.isAmrap && (
-          <Typography variant="body2" color="primary">
+          <Typography variant="body2" color="secondary">
             (AMRAP)
           </Typography>
         )}
       </FormLabel>
-      <ToggleButtonGroup
```

Comment:
This part of the diff is a bit messy, but it's just because I needed to wrap the existing ToggleButtonGroup in row stack so I could render the AMRAP button to the left, separate from the other ones.